projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -181,14 +183,41 @@ export class RootComponent implements OnInit, OnDestroy {
     this.route.queryParamMap.subscribe(query => {
       const version = query.get('version');
       const sheet = query.get('sheet');
+      const selectedOrgans = localStorage.getItem('selectedOrgans');
```

Comment:
Do not save to localStorage. This should just be instance memory (ie if they refresh, they start over). You may be able to store the selected organs in the route/get parameter, which would be ideal.